functions/services.py

```diff
@@ -0,0 +1,50 @@
+import csv, os
+from dotenv import load_dotenv
+from classes.fetcher import Fetcher
+from classes.fetch_services import YouTubeFetchService, YtDlpFetchService
+from classes.caching import FileCache
+from classes.printers import ConsolePrinter
+from functions.messages import suc, inf, err
+
+# Load environment configuration from a `.env` file if present.
+load_dotenv()
+
+API_KEY = os.getenv("apikey")  # may replace this
+
+def get_fetcher() -> Fetcher:
+    """Return the standard video fetcher used by the Top 10 Pony Videos
+    applications (currently configured for YouTube and yt-dlp)."""
+
+    inf("* Configuring video data fetcher...")
+    fetcher = Fetcher()
+    fetcher.set_printer(ConsolePrinter())
+
+    # Set up a cache file for video data.
+    response_cache_file = os.getenv("response_cache_file")
+
+    if response_cache_file is not None:
+        inf(f"  * Fetched video data will be cached in {response_cache_file}.")
+        fetcher.set_cache(FileCache(response_cache_file))
+
+    # Configure fetch services. Currently the YouTube Data API and yt-dlp are
+    # supported.
+    inf("  * Adding fetch services...")
+    accepted_domains = []
```

Comment:
Before merging to master, replace lines 32-35 with


    accepted_domains = load_text_data('data/accepted_domains.txt')

Also import `load_text_data` from functions.general